nilearn/plotting/tests/test_find_cuts.py

```diff
@@ -94,3 +95,69 @@ def test_tranform_cut_coords():
     for direction in 'xyz':
         assert_equal(len(_transform_cut_coords(cut_coords, direction, affine)),
                      n_cuts)
+
+
+def test_find_parcellation_atlas_cut_coords():
+    data = np.zeros((250, 250, 250))
```

Comment:
You use relatively large numbers. hlaving them or even dividing by 4 or 5 would save time.